web/src/layout/Header/navbar/Debug.tsx

```diff
@@ -54,7 +59,7 @@ const Phase = () => {
   const { data: phase } = useSortitionModulePhase({
     watch: true,
   });
-  return <>{phase !== undefined && <label>Phase: {Phases[phase]}</label>}</>;
+  return <>{phase !== undefined && <StyledLabel>Phase: {Phases[phase]}</StyledLabel>}</>;
```

Comment:
Lately I have been trying to use the following syntax for this situations:
```suggestion
  return (phase !== undefined ? <StyledLabel>Phase: {Phases[phase]}</StyledLabel> : null);
```
(also, try using the `isUndefined` function from `utils/index.ts`)